contrib/kind.sh

```diff
@@ -497,7 +499,7 @@ build_ovn_image() {
 }
 
 create_ovn_kube_manifests() {
-  pushd ../dist/images
+  pushd ${DIR}/../dist/images
   ./daemonset.sh \
```

Comment:
here?

Reply:
The same as above. I think it'd be best to have everything absolute..

Reply:
`${DIR}/daemonset.sh`  ?

Reply:
the daemonset.sh is in ${DIR}/../dist/images, though. I think doing the pushd and then simply ./daemonset.sh is ok, I also addressed the same around line 493